app/pages/sections/catalogue-solutions/order-item/manifest.json

```diff
@@ -37,6 +37,35 @@
       }
     }
   },
+  "columnInfo": [
```

Comment:
can you nest columnInfo, data and columnClass into another object please `addPriceTable` perhaps